api/src/controllers/young/index.js

```diff
@@ -464,6 +464,27 @@ router.put("/", passport.authenticate("young", { session: false, failWithError:
   }
 });
 
+router.put("/:id/change-cohort/:newCohort", passport.authenticate("young", { session: false, failWithError: true }), async (req, res) => {
```

Comment:
on peut ne pas spécifier la cohorte directement dans la route, on peut la passer dans le body (avec le reste des infos d'ailleurs)
```suggestion
router.put("/:id/change-cohort", passport.authenticate("young", { session: false, failWithError: true }), async (req, res) => {
```